ironmon_tracker/Utils.lua

```diff
@@ -29,6 +29,12 @@ function Utils.centerTextOffset(text, charSize, width)
 	return (width - (charSize * text:len())) / 2
 end
 
+-- Accepts a positive or negative integer and returns a string formatted as "12,345"
+function Utils.formatNumberWithCommas(integer)
+	if integer == nil then return "" end
+	return tostring(math.floor(integer)):reverse():gsub("(%d%d%d)","%1,"):gsub(",(%-?)$","%1"):reverse()
```

Comment:
If the intent is to provide specifically an integer, why does it need to be floor rounded? 
The way this is written seems more like "Accepts a number, converts to integer, and returns a string formatted as '12,345'"

Reply:
Good note. I simply borrowed the code solution from somewhere online. I tested it with an input of `1234.56` and removed the `math.floor()` call, seems to work just fine. So I'll update it as such.

Reply:
There were some issues with getting it to be compatible with fractional numbers, so I found a different solution that accounts for a wider selection of inputs.